src/test/java/edu/ucsb/mapache/entities/StudentTests.java

```diff
@@ -0,0 +1,100 @@
+package edu.ucsb.mapache.entities;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import org.junit.jupiter.api.Test;
+
+public class StudentTests {
+  @Test
+  public void testStudent_notEqualNull() throws Exception {
+    Long id = 1L;
+    Student student = new Student(id, "a@gmail.com", "team1");
+    assertNotEquals(student, null);
+  }
+
+  @Test
+  public void testStudent_notEqualAnotherClass() throws Exception {
+    Long id = 1L;
+    Student student = new Student(id, "a@gmail.com", "team1");
+    assertNotEquals(student, new Object());
+  }
+
+  @Test
+  public void testStudent_equalsSelf() throws Exception {
+    Long id = 1L;
+    Student student = new Student(id, "a@gmail.com", "team1");
+    assertEquals(student, student);
+  }
+
+  @Test
+  public void testStudent_toString() throws Exception {
+    Long id = 1L;
+    Student student = new Student(id, "a@gmail.com", "team1");
+    assertEquals(student.toString(), String.format("Student[ id=%d, email=%s, teamName=%s]", id, "a@gmail.com", "team1"));
+  }
+  @Test
+  public void testStudent_equalsSelf_With_Comparison_to_Other_Student_Objects() throws Exception {
+    long id1 = 1L;
+    long id2 = 2L;
+    long id3 = 3L;
+    Student student = new Student(id1, "email", "team");
+    Student student2 = new Student(id2, "email", "team2");
+    Student student3 = new Student(id3, "email3", "team3");
```

Comment:
instead of having to define the longs separately, can we define them inline? 

example: `new Student(1L, "email", "team")`